tests/CompilerTest.php

```
@@ -78,8 +78,8 @@ public function testParentClassName(object $class): void
     /** @depends testNewInstance */
     public function testBuildClassWeaved(FakeMock $weaved): void
     {
-        assert(isset($weaved->bindings));
-        $weaved->bindings = $this->bind->getBindings();
+        assert(property_exists($weaved, 'state'));
+        $weaved->state->bindings = $this->bind->getBindings();
```

PHPStan (2.0.3)

Cannot access property $bindings on mixed.


(property.nonObject)